tests/buyers/views/test_buyers.py

```diff
@@ -22,38 +22,115 @@
 po = functools.partial(mock.patch.object, autospec=True)
 
 
+@pytest.fixture()
+def find_briefs_mock():
+    base_brief_values = {
+        "createdAt": "2016-02-01T00:00:00.000000Z",
+        "frameworkSlug": "digital-outcomes-and-specialists",
+        "lot": "digital-specialists"
+    }
+
+    find_briefs_response = {
```

Comment:
Good, sensible idea to have one brief of every status which was lacking in existing tests.